pandas/core/arrays/categorical.py

```diff
@@ -457,6 +457,9 @@ def _formatter(self, boxed=False):
         # Defer to CategoricalFormatter's formatter.
         return None
 
+    def _formatting_values(self):
```

Comment:
`_formatting_values` is currently deprecated on EAs, so why was this needed?

Reply:
to support the recursive behaviour and remove the special casing.